aptos-move/framework/aptos-framework/sources/datastructures/big_ordered_map.move

```diff
@@ -215,6 +241,22 @@ module aptos_std::big_ordered_map {
         self.nodes.allocate_spare_slots(num_to_allocate)
     }
 
+    /// Returns true iff the BigOrderedMap is empty.
+    public fun is_empty<K: store, V: store>(self: &BigOrderedMap<K, V>): bool {
+        let node = self.borrow_node(self.min_leaf_index);
+        node.children.is_empty()
+    }
+
+    /// Returns the number of elements in the BigOrderedMap.
+    /// This is an expensive function, as it goes through all the leaves to compute it.
+    public fun compute_length<K: store, V: store>(self: &BigOrderedMap<K, V>): u64 {
```

Comment:
intentionally not calling length() because it is expensive to call. is that reasonable